fedimint-testing/src/btc/mock.rs

```diff
@@ -65,7 +68,7 @@ struct FakeBitcoinTestInner {
     /// Simulates the merkle tree proofs
     proofs: BTreeMap<Txid, TxOutProof>,
     /// Simulates the script history
-    scripts: BTreeMap<ScriptBuf, Vec<Transaction>>,
+    scripts: BTreeMap<ScriptBuf, Vec<(Transaction, u64)>>,
```

Comment:
Oh, it's a block height? Hard to tell from looking at it. Either a comment, or maybe

`type BlockHeight = u64;` and `scripts: BTreeMap<ScriptBuf, Vec<(Transaction, BlockHeight)>>`?